opus/application/apps/search/forms.py

```diff
@@ -114,11 +115,23 @@ def __init__(self, *args, **kwargs):
                 label = 'max' if num == '2' else 'min'
 
                 pi = get_param_info_by_slug(slug, 'search')
+                # placeholder for input hints (only apply to Min input for now)
+                hints = pi.field_hints1 if pi.field_hints1 else ''
 
                 self.fields[slug] = MultiFloatField(
-                     required=False,
-                     label = label,
-                     widget = forms.TextInput(attrs={'class':label + ' RANGE'}),
+                    required=False,
+                    label = label.title(),
```

Comment:
What is the "title()" for?

Reply:
Change to capitalize(), this is used to change "min" & "max" to "Min" & "Max".